core/pom.xml

```diff
@@ -105,12 +101,12 @@
         <dependency>
             <groupId>org.java-websocket</groupId>
             <artifactId>Java-WebSocket</artifactId>
-            <version>1.3.8</version>
+            <version>1.4.0</version>
```

Comment:
The version number can be added as a property.

Reply:
fixed